readme.txt

```diff
@@ -33,5 +33,38 @@ This theme bundles the following third-party resources:
 
 === Fonts ===
 
+Fira Code Font
+Copyright (c) 2014, The Fira Code Project Authors
+License: SIL Open Font License, 1.1, https://opensource.org/licenses/OFL-1.1
+Source: https://github.com/tonsky/FiraCode
+
+Manrope Font
+Copyright (c) 2018 The Manrope Project Authors 
+License: SIL Open Font License, 1.1, https://opensource.org/licenses/OFL-1.1
+Source: https://github.com/sharanda/manrope
+
+Vollkorn Font
+Copyright (c) 2005–2018, Friedrich Althausen 
+License: SIL Open Font License, 1.1, https://opensource.org/licenses/OFL-1.1
+Source: https://github.com/FAlthausen/Vollkorn-Typeface
+
+Fira Sans Font
+Digitized data copyright (c) 2012-2015, The Mozilla Foundation and Telefonica S.A.
+License: SIL Open Font License, 1.1, https://opensource.org/licenses/OFL-1.1
+Source: https://github.com/mozilla/Fira
+
+Platypi Font
+License: SIL Open Font License, 1.1, https://opensource.org/licenses/OFL-1.1
+Source: https://github.com/d-sargent/platypi
+
+Ysabeau Office Font
+License: SIL Open Font License, 1.1, https://opensource.org/licenses/OFL-1.1
+Source: https://github.com/CatharsisFonts/Ysabeau
```

Comment:
Were the fonts downloaded from the GitHub repositories?
I am asking because sometimes the version on GitHub is not the same as on Google Fonts. Sometimes Google Fonts are not up to date.

Reply:
I downloaded this font from https://fonts.google.com/ 

So, do you want me to replace the font from GitHub?

Reply:
No we only need to do that if the Google font version is missing glyphs / have bugs. 

The URL can be kept in the readme, but lets not call it "source" if it is not the source, because it can get confusing when trying to update the fonts later.

So I think the file can include both the GitHub link for reference, and the google fonts link as the source?

Reply:
Thank you, that sounds good. I think we can do in this way. 

Example,

```
Fira Code Font
Copyright (c) 2014, The Fira Code Project Authors
License: SIL Open Font License, 1.1, https://opensource.org/licenses/OFL-1.1
Reference: https://github.com/tonsky/FiraCode 
Source: https://fonts.google.com/specimen/Fira+Code

```

Can you verify if this is the right way to do?